### PR TITLE
chore: fix gcloud install with latest ubuntu

### PR DESCRIPTION
**Pull request by ericzzzzzzz**
Fixes: #8137  

updating strategy matrix causes ci jobs stuck.. that might also happen if we update top level info in workflow file, I think we probably need admin power to merge this pr. #8117 has the same issue,

### CODE REVIEW

**Comment by ericzzzzzzz** [comment]
- updating strategy matrix causes ci jobs stuck.. that might also happen if we update top level info in workflow job, I think we probably need admin power to merge this pr.

**Comment by ericzzzzzzz** [comment]
Hi @tejal29  can I get your 👀  on this pr ? The some github checks got stuck due to changes made in workflow file, I think I may need you to approve and merge it. @aaron-prindle approved this before, but I was not able to merge it :(

**Comment by aaron-prindle** [comment]
I also can't merge this with the current Github settings ( I believe I have the same access)

**Review by aaron-prindle — APPROVED** [review]

**Pull request merged**
